### ISSUE TITLE
[BUG] Vatek USB DVB-T/T2 errors

### ISSUE THREAD

**Issue by skunge2000** [tsduck/tsduck#1429]
#### Bug description:
Error when trying to use VATEK USB DVB-T/T2 modulator 

#### How to reproduce:
I have installed tsduck from the latest release tsduck_3.36-3528.ubuntu23_arm64.deb 
I'm running Ubuntu 23.10 on a Raspberry pi 4
This was a clean build and this was the first thing I attempted to do

When I run tsvatek to detect vatek devices I get this

```
[0000694c:error   ] - usb_api_ll_enum_common   - PID : 0x00002c42 ; VID :0x00001031

[0000694d:error   ] - cross_devices_create     - index : 0

Device 0: .//3-0//vatek-usb
 - [status              ] : select   - waitcmd
 - [errcode             ] : uint32   - 0x00000000
 - [chip_module         ] : select   - a3
 - [hal_service         ] : select   - transform
 - [version             ] : uint32   - 0x02050211
 - [peripheral_en       ] : flags    - 0x01000102
 - [input_support       ] : flags    - 0x00370007
 - [output_support      ] : flags    - 0x0000ff03
```
And I get screens full of errors when I try to modulate a transport stream using a script that previously worked.

#### Expected behavior:
I would not expect the errors

**Could this be a hardware fault with my VATEK modulator?**

#### Environment:
 - OS: Linux
 - OS version: Ubuntu 23.10 (6.5.0-1009-raspi)
 - TSDuck full version: 3.36-3528
 - Installation type: official binary release downloaded as .deb from GitHub releases

**Comment by lelegard** [comment]
@a83912a, Richie, could you have a look on this one, please ?

**Comment by a83912a** [comment]
@skunge2000 
Based on the provided log, it seems that VATEK USB was found and executed successfully. The error mentioned above appears to be a warning added during the initial validation and may not indicate an actual error. Could you confirm if VATEK is producing the correct output?

**Comment by skunge2000** [comment]
@a83912a I get no output and the following errors :

```
[00077bd1:error   ] - usb_api_ll_enum_common   - PID : 0x00002c42 ; VID :0x00001031

[00077bd2:error   ] - cross_devices_create     - index : 0

[00077fd0:error   ] - usbstream_async_handler  - slice pustream->stream_packets : 128

[00077fd0:error   ] - usbstream_async_handler  - slice pustream->stream_packets : 128

[00077fd0:error   ] - usbstream_async_handler  - slice pustream->stream_packets : 128
```

The usbstream_async_handler error continues very frequently.

**Comment by a83912a** [comment]
@skunge2000 
I tested it, and the error message indicates the remaining space in the VATek USB packet. The VATek USB continues to function properly; it's just that this message keeps appearing frequently. I will fix this issue in the next version. Since I'm about to take a long break, it will take some time before I update the VATek SDK. Thank you.

**Comment by skunge2000** [comment]
OK - I'll continue testing and ignore the messages. Something else must have broken with my install - I didn't remember the error messages previously so assumed they were related. 
(If the errors aren't actually errors then calling them errors is a bit confusing :) )

**Comment by lelegard** [comment]
@skunge2000, @a83912a, what is the status of this issue? Can we consider it as closed or do we expect additional tests or feedbacks?

**Comment by a83912a** [comment]
@skunge2000 
I have updated the VATek binary to the corrected version. You can simply recompile the TSDuck source code to use it. Thank you.

@lelegard 
If no issues are reported, we can close this issue.

**Comment by lelegard** [comment]
Thank you @a83912a. So the fix is in libvatek 3.10, right?
